project_report/tex/requirements.tex

```diff
@@ -8,6 +8,29 @@ \subsubsection{Functional \& Non-Functional Requirements}
 
 \subsubsection{The Physical Setup (The Brewery Machine)}
 
+The machine is able to produce a different kind of beers depending on the recipe
+used. The beer is produced in batches which have their own batch id. When 
+producing a batch the quantity, speed and batch id can be configured by an 
+external system. When a batch is done the quality inspection system measures 
+the quality for future use. These values are all accessible in the machines 
+SCADA system that monitor and supervises both the machines PLC system and the 
+internal sensors PLC system as well as the quality inspection system.
+
+\myparagraph{Sensors}
+To ensure the most optimal environment the machine is equipped with three 
+sensors that measure while production is happening. This data is useful because 
+it gives insight into the internal environment of the machine while producing 
+products. Currently, it measures temperature, humidity and vibration. On that 
+note is important to remember the machine itself is not able to collect these 
+data and are collected by an external system.
+
+\myparagraph{Quality Inspection System}
+In order to make it possible to calculate an error function, the system is 
```

Comment:
Not sure if this is taking the meaning out of the paragraph?
In order to make it possible to calculate an error function is used. The system is equipped with a sophisticated quality inspection system. 

Maybe like this not sure?: 
The system is equipped with a sophisticated quality inspection system, that allow it to calculate errors?

Reply:
it is the inspection that makes it possible to calculate the error function